charts/radar-output/templates/configmap-restructure.yaml

```diff
@@ -114,9 +114,9 @@ data:
       # Output directory
       output: "{{ .Values.paths.output }}"
       # Output path construction factory
-      # factory: org.radarbase.output.path.MonthlyObservationKeyPathFactory
+      factory: {{ .Values.paths.factory }}
       # Additional properties
-      # properties: {}
+      properties: {{ .Values.paths.properties | toYaml | indent 8 | trim }}
```

Comment:
I'm not sure if this is the correct format, I always struggle and try a few things until one of them works. You can use `helmfile template` and see if the output is in correct format. 
I usually add a `|` and put them in a new line, might need to use `nindent` as well.

Reply:
@K1Hyve I have tested the latest push with helmfile and it works.
There is already a `|` on the top of the yaml file. Moving the values expression to a new line fixes the issues.